frontend/src/main/components/Sections/SectionsTable.js

```diff
@@ -1,67 +1,109 @@
-import OurTable from "main/components/OurTable";
+import SectionsTableBase from "main/components/SectionsTableBase";
 
 import { yyyyqToQyy } from "main/utils/quarterUtilities.js";
-import { convertToFraction, formatDays, formatInstructors, formatLocation, formatTime, isSectionAsString } from "main/utils/sectionUtils.js";
+import { convertToFraction, formatDays, formatInstructors, formatLocation, formatTime, isSection, _isSectionAsString} from "main/utils/sectionUtils.js";
```

Comment:
If you're going to rename this function, can you remove the old import if it doesn't lead to anything anymore?

```suggestion
import { convertToFraction, formatDays, formatInstructors, formatLocation, formatTime, isSection } from "main/utils/sectionUtils.js";
```